pocketlint/tests/test_restructuredtext.py

```diff
@@ -343,6 +347,39 @@ def test_check_section_delimiter_bad_marker_length(self):
         self.assertEqual(expect, self.reporter.messages)
         self.assertEqual(1, self.reporter.call_count)
 
+    def test_semantic_newline_fullstop(self):
+        """Check for a sentence with a . and no new line."""
+        content = (
+            'Sentence. New sentence\n'
+            )
+        checker = ReStructuredTextChecker('bogus', content, self.reporter)
+        checker.check_semantic_newline()
+        expect = [('Newline not created after a sentence.')]
```

Comment:
what is the point of these parenthesis here?

I know that existing tests are not a good example, but for new tests please consider using the structure recomended by the styleguide http://styleguide.chevah.com/testing.html#structure-of-a-test